public/data/ann-design-max.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "Anastestia Onyekaba",
+  "type": "personal",
+  "bio": "UI/UX Designer | Frontend Developer | Co-Founder of N&S Tech | Open Source Enthusiast!",
+  "avatar": "https://github.com/Ann-design-max.png",
+  "links": [
+    {
+      "name": "Follow me on Github ",
```

Comment:
I recommend capitalizing the `h` here. What do you think? 😄
```suggestion
      "name": "Follow me on GitHub",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
@Panquesito7 Thank you for pointing this out. It has been resolved in this commit https://github.com/EddieHubCommunity/LinkFree/pull/1730/commits/6507b212b6485bd8b4be8654194b1acbc3ebdf51